.github/workflows/windows.yml

```diff
@@ -100,32 +87,112 @@ jobs:
         run: |
           choco install --no-progress memurai-developer
 
-      - name: Run tests (minimal extensions)
-        if: always() && steps.setup.outcome == 'success'
+      - name: Run tests
         run: |
           $env:Path = 'c:\php;' + $env:Path
-          $env:SYMFONY_PHPUNIT_SKIPPED_TESTS = 'phpunit.skipped'
           $x = 0
 
-          Copy c:\php\php.ini-min c:\php\php.ini
           Remove-Item -Path src\Symfony\Bridge\PhpUnit -Recurse
           mv src\Symfony\Component\HttpClient\phpunit.xml.dist src\Symfony\Component\HttpClient\phpunit.xml
-          php phpunit src\Symfony --exclude-group tty,benchmark,intl-data,network,transient-on-windows || ($x = 1)
+          php phpunit src\Symfony --exclude-group tty --exclude-group benchmark --exclude-group intl-data --exclude-group network --exclude-group transient-on-windows || ($x = 1)
           # HttpClient tests need to run separately, they block when run with other components' tests concurrently
           php phpunit src\Symfony\Component\HttpClient || ($x = 1)
 
           exit $x
 
+  windows-all-extensions:
+    name: x86 / all extensions / lowest-php
+
+    defaults:
+      run:
+        shell: pwsh
+
+    runs-on: windows-2022
+
+    env:
+      COMPOSER_NO_INTERACTION: '1'
+      SYMFONY_DEPRECATIONS_HELPER: 'strict'
```

Comment:
this env var should not be necessary anymore, as it won't be used with PHPUnit 11

Reply:
removed it